include/FockSpace/FockSpace.hpp

```diff
@@ -220,6 +220,39 @@ class FockSpace: public BaseFockSpace, public FockPermutator<FockSpace> {
      */
     VectorX<double> evaluateOperatorDiagonal(const SQHamiltonian<double>& sq_hamiltonian) const override;
 
+    /**
+     *  Evaluate a one electron operator in a matrix vector product
+     *
+     *  @param one_op                       the one electron operator expressed in an orthonormal basis
+     *  @param x                            the vector upon which the evaluation acts 
+     *  @param diagonal                     the diagonal evaluated in the Fock space
+     *
+     *  @return the one electron operator's matrix vector product in a vector with the dimensions of the Fock space
+     */
```

Comment:
```suggestion

     */
```

Also: apply changes in the same train of thought for all analogous function calls